conf-sample.js

```diff
@@ -241,4 +241,5 @@ c.output  = {}
 // REST API
 c.output.api = {}
 c.output.api.on = true
+c.output.api.ip = // either a IPv4 or IPv6 address (as a string) to listen on, uses all available interfaces if omitted
```

Comment:
shouldn't this be an empty string, or null?

Reply:
I want to avoid breaking existing installations thus its undefined as api.js takes care of it anyway.

Reply:
I commented the same thing but realized api.js handled it as well.  I think that if it was set to "0.0.0.0" it wouldn't break existing installations as well as function as an opt-in parameter that the user has to change in their own conf.js

Reply:
Sounds like the most elegant solution as it also make it clear how the settings is supposed to look like.